src/docs/contributing/pages/components.mdx

```diff
@@ -27,7 +27,16 @@ Attributes:
 - `level` (string)
 - `dismiss` (boolean)
 
-See also the Note component.
+Use this for these types of content:
+
+- Important: Use these for information that's critical to know; it's important for users to read, but won't cause a catastrophic problem if they don't read it. These include versions that introduce breaking changes or feature limitations. Use infrequently.
+  - Use this with or without a title
+- Warning: Use these for items that MUST be well understood before proceeding. These highlight information that could cause users to make catastrophic errors that break their applications in ways that are very difficult to fix or create liabilities for them, such as information needed to avoid leaking PII. These should be used very rarely.
```

Comment:
```suggestion
- **Warning:** Use these for items that MUST be well understood before proceeding. These highlight information that could cause users to make catastrophic errors that break their applications in ways that are very difficult to fix or create liabilities for them, such as information needed to avoid leaking PII. These should be used very rarely.
```